Added the Introduction to the Information Security and Quality Assura… (#35576)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md

---
## Introduction to the Information Security and Quality Assurance Projects

> Now it’s time to put your newly learned skills to work! By working on these projects, you would have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.

Reviewer: ```suggestion
Now it’s time to put your newly learned skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

> So, now it’s time to put your newly learnt skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.

Reviewer 2: ```suggestion
Now it’s time to put your newly learnt skills to work! By working on these projects, you would have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

Reviewer 3: Please don't use the word `learnt`.  That is British English and it should be the American English word `learned`.

Author: Yes, I have made the cahnges.

Reviewer 2: I don't see the changes though

Author: I updated the file can you check this one? @thecodingaviator

In this section you get the chance to:
* Build a Metric-Imperial Converter
* Build an Issue Tracker
* Build a Personal Library
* Build a Stock Price Checker
* Build an Anonymous Message Board

> By the end of this, you will have five IS & QA projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer 2: ```suggestion
When you are done, you will have five Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

> When you are done, you will have five Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer 2: Do we want to list the number of challenges over here>

cc: @RandellDawson @moT01

Reviewer 4: It says there's five projects, I don't like to be repetitive - what did you have in mind?

Reviewer 2: I want to get the number removed completely. See https://github.com/freeCodeCamp/freeCodeCamp/pull/34405#discussion_r264282065

Reviewer 4: Sounds good to me either way, but I suppose that's a good point - if the number ever changes, we don't have to modify these.

Reviewer 2: @Sumer16 Go ahead and reword this so it doesn't show no of challenges

> When you are done, you will have plenty of Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer: ```suggestion
When you are done, you will have plenty of Information Security & Quality Assurance projects under your belt along with a certification that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
